mkosi/__init__.py

```diff
@@ -301,6 +303,26 @@ def print_running_cmd(cmdline: Iterable[str]) -> None:
     "armhfp": "armhf",
 }
 
+# Gentoo globals
+try:
+    from portage.package.ebuild.config import config  # type: ignore
```

Comment:
Can we just import this inside the function where we use it?

Reply:
done